### PR TITLE
Move Look & feel section to Shields prefs page

### PR DESCRIPTION
**Pull request by simonhong**
Look & feel section is moved to the top to include choosing Shields default view.

Resolves https://github.com/brave/brave-browser/issues/10502

Screen Shot 2020-06-29 at 9 53 07 AM

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [x] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

**Comment by petemill** [comment]
If this can wait until after cr84 has landed in master, that would be great.

**Comment by simonhong** [comment]
@petemill can wait :)

**Comment by simonhong** [comment]
@petemill Rebased on cr84 :)

File: browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html [hunk 1]
```diff
@@ -138,13 +149,7 @@
...
       </select>
     </div>
-    <div class="settings-box">
-      <div class="label shields-primary-title">$i18n{shieldsLookFeelTitle}</div>
-    </div>
-    <settings-toggle-button
-        class="continuation"
-        pref="{{prefs.brave.shields.stats_badge_visible}}"
-        label="$i18n{showStatsBlockedBadgeLabel}">
+
```
[the remark this answers is not in the document]
Comment (simonhong) [reply browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html]: removed.

File: browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html [hunk 2]
```diff
@@ -34,10 +34,13 @@
...
       }
-       .shields-view-label {
-         margin: 0 20px;
-         cursor: pointer;
-       }
+      .shields-view-label {
+        margin: 0 20px;
+        cursor: pointer;
+      }
+      .default-view-padding {
```
[the remark this answers is not in the document]
Comment (simonhong) [reply browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html]: All done.

**Comment by simonhong** [comment]
@petemill Thanks for review!

**Comment by simonhong** [comment]
Merged because only test-install step is failed.

**Pull request merged**